types/flowjs__flow.js/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": [true, { "mode": "code", "errors": [["NeedsExportEquals", false]] }]
```

Comment:
why this exclusion? It's like UMD, with exports and namespace export `Flow`
Using UMD pattern should work here I think

Reply:
I've added the exclusion because when I tried to write
```typescript
export = Flow;
```
instead of
```typescript
export default Flow;
```
there was a compile error `An export assignment cannot be used in a module with other exported elements.`

Reply:
I've missed the part this is re-export. 
This works perfecty well locally:
```ts
import flowjs = require('flowjs');

declare module 'flowjs' {
    // tslint:disable:no-misused-new
    interface Flow {
        new (options?: FlowOptions): Flow;
    }
}

declare const Flow: flowjs.Flow;
export = Flow;
export as namespace Flow;
```
note it augments flow.js to add proper support for constructor like usage via `Flow`:
https://github.com/flowjs/flow.js#how-can-i-use-it

Reply:
But then it doesn't export what it should export
![изображение](https://user-images.githubusercontent.com/1297574/133344696-a8cc3669-6be4-4e3d-a59a-34fe5ba1db3e.png)

Reply:
You have to access it under `Flow.FlowOptions` (if you’re combining with the default import).

Reply:
That doesn't work either
![изображение](https://user-images.githubusercontent.com/1297574/133350765-9055438d-0fa6-4cea-86bb-2cb008dc6b6d.png)

Reply:
no, it won't solve all the problems. Just pointed the very first user contact with package is not supported at all (readme-based development).
The proper fix is to correct `flow.js` exports and re-export properly here. For now that's OK I'say.